crates/abq_cli/src/args.rs

```diff
@@ -318,6 +318,18 @@ pub enum Command {
         #[clap(long, default_value = "7")]
         batch_size: NonZeroU64,
 
+        // 🤔 right now this feels like more of a flag than a config option. Should we just make it a flag? `--group-by-file`?
+        // I can imagine expanding on this in the future, but maybe then we can make it an option.
```

Comment:
+1 for keeping it a config option for future-proof reasons. Being explicit about the configuration is also nice